src/pages/WishListPage/WishListPage.jsx

```diff
@@ -12,117 +12,170 @@ import {
       product15
   } from "./../../../src/assets/images/Products/Products";
 import { useAuth } from '../../context/AuthContext';
+import axios from 'axios';
+import WishListsCards from '../../components/UI/Card/WishListCards/WishListsCards';
+import { useWishList } from '../../context/WishListContext';
+
+
 function WishListPage() {
     const { login } = useAuth();
+    const [WishListProductList,setWishListProductList] = useState([]);
+    const {WishListState,setWishListState} =useWishList();
     console.log("auth login ",login);
+    console.log("login auth",login);
+    useEffect(()=>{
+        try{
+            (() => {
+                var res = axios.get("/api/user/wishlist",{ headers:{
```

Comment:
Suggestion: we can use let instead of var unless needed specifically.

Reply:
ok done